packaging/install/cluster-operator/040-Crd-kafka.yaml

```diff
@@ -2039,6 +2039,27 @@ spec:
                                   `Deployments`, `Pods`, and `Services`.
                             description: Metadata applied to the resource.
                         description: Template for the Kafka service account.
+                      jmxSecret:
+                        type: object
+                        properties:
+                          metadata:
+                            type: object
+                            properties:
+                              labels:
+                                x-kubernetes-preserve-unknown-fields: true
+                                type: object
+                                description: Labels added to the resource template.
+                                  Can be applied to different resources such as `StatefulSets`,
+                                  `Deployments`, `Pods`, and `Services`.
```

Comment:
In all the CRDs, you seem to have the descriptions in Helm Chart as single line and in install as multiline. Did you changed that manually? Or is this how it was generated by the make commands? If you changed it manually, it would fail in the build, the files need to be as they were generated. (if it was generated automatically like this, then please just ignore this comment)